### PR TITLE
Added simplecov closes #2

### LINKED ISSUE #2: Chore: Implement SimpleCov
Simplecov or other?

### PR DESCRIPTION
**Pull request by mikedao**
Made appropriate changes to add Simplecov reporting to our application. Also made some improvements to code I saw while I was working on it, detailed below.

* Added coverage folder to local .gitignore so we would not sync the SimpleCov reports
* Added simplecov to Gemfile, also found that Gemfile had a second gem 'pry' - removed it.
* Gemfile.lock was updated because I ran bundle install to ensure it was working.

All tests pass, however, I encountered errors reseeding the database. We should leave this as is and fix when seed.rb file is addressed in #41

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,6 @@
 /log/*
 !/log/.keep
 /tmp
+
+# Ignore Simplecov coverage report
```
Comment (EmilyMB) [inline .gitignore]: Do we need the comment in here on 19? I think it's pretty self-explanatory.

File: Gemfile [hunk 2]
```diff
@@ -58,6 +59,5 @@ group :development, :test do
   gem 'spring'
   gem 'capybara'
   gem 'mocha'
-  gem 'pry'
   gem 'launchy'
 end
```
Comment (EmilyMB) [inline Gemfile]: Good job on catching the duplicate pry :smile: 
How about going ahead and setting versions for all the :gem: s now?

File: bin/rails [hunk 3]
```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env ruby
+if ENV['RAILS_ENV'] == 'test'
+  require 'simplecov'
+  SimpleCov.start 'rails'
+  puts "required simplecov"
```
Comment (EmilyMB) [inline bin/rails]: I guess this is standard? Seems like extraneous output to me... :confused:

File: test/test_helper.rb [hunk 4]
```diff
@@ -1,3 +1,5 @@
+require "simplecov"
+SimpleCov.start
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
```
Comment (EmilyMB) [inline test/test_helper.rb]: :tada: It will be nice to start using this :sparkler:

File: Gemfile (hunk 2 above)
Comment (mikedao) [inline Gemfile]: Is that convention to do so?

File: bin/rails (hunk 3 above)
Comment (mikedao) [inline bin/rails]: It might be, but I just followed documentation. 

The thing is that it doesnt actually out put that string, so maybe its needed for something?

File: test/test_helper.rb (hunk 4 above)
Comment (mikedao) [inline test/test_helper.rb]: We actually have great coverage, I think we should talk about how we can get it to 100 percent tomorrow.

File: .gitignore (hunk 1 above)
Comment (mikedao) [inline .gitignore]: I wanted to have the comment so it would match the rest of the .gitignore file. I think it looks better this way.

File: Gemfile (hunk 2 above)
Comment (EmilyMB) [inline Gemfile]: So, I found this commentary on it: http://robots.thoughtbot.com/a-healthy-bundle
TL;DR: unless you are sure the gem has an excellent history of backwards compatibility (e.g. pg), it is a good idea to use "pessimistic versioning"--e.g. gem 'nokogiri', '~> 1.0.3'.

File: .gitignore (hunk 1 above)
Comment (EmilyMB) [inline .gitignore]: Fair enough. Thanks for explaining!

File: bin/rails (hunk 3 above)
Comment (EmilyMB) [inline bin/rails]: Probably just advertising their :gem: ! We can remove it later if it isn't adding value :moneybag:

File: test/test_helper.rb (hunk 4 above)
Comment (EmilyMB) [inline test/test_helper.rb]: And improving the tests that are there :smiling_imp:

File: bin/rails (hunk 3 above)
Comment (mikedao) [inline bin/rails]: Is it really advertising if it never appears in the server logs?

**Comment by mikedao** [comment]
Added strict gem versioning.
